server.js

```diff
@@ -1,4 +1,4 @@
-// TASK 1: Setting up server
```

Comment:
The comment should use a consistent format. Instead of 'TATASKSK 1:', it should be 'TASK 1:' for better readability.